### PR TITLE
humann3

### PR DESCRIPTION
**Pull request by kohjy-ag**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/humann/main.nf
Comment (nvnieuwk) [inline tests/modules/humann/main.nf:11]: I would suggest using the `params.test_data['homo_sapiens']['illumina']['test_1_fastq_gz']` here instead of the full link. You can find all test datasets for modules either on https://github.com/nf-core/test-datasets/tree/modules or in the `conf/test_data.config`

File: tests/modules/humann/main.nf
Comment (nvnieuwk) [inline tests/modules/humann/main.nf:12]: What are these files meant to be?

File: tests/modules/humann/test.yml
Comment (nvnieuwk) [inline tests/modules/humann/test.yml:6]: Remove versions.yml from the file, if you update to the latest version of `nf-core` you won't have to delete these manually (you can also see that you are not getting any other output, is this supposed to happen?)

File: modules/humann/meta.yml
Comment (nvnieuwk) [inline modules/humann/meta.yml:25]: Update your input fields to correspond with the actual input of the module :)

File: modules/humann/meta.yml
Comment (nvnieuwk) [inline modules/humann/meta.yml:45]: Also update your output field to correspond with the real output :)

**Review by nvnieuwk — CHANGES_REQUESTED** [review]

**Comment by nvnieuwk** [comment]
You can also see there are some trailing whitespaces detected by the `Code Linting / EditorConfig` check. These should be removed too :)

**Comment by JoseEspinosa** [comment]
Hi @kohjy-ag , thank you for this PR! We are merging as many modules as possible now due to and impending restructuring of the entire repo. This will mean you will need to update the module to reflect these changes before it can be merged in the future. It appears like this module isn't ready to be merged so we are converting it to draft and adding the WIP label. If this isn't the case please let us know and we will try to get the module in before the changes. Thanks again!

**Comment by matthdsm** [comment]
Hi there!

We've noticed there hasn't been much activity here. Are you still planning on working on this?
If not, you can ignore this message and we'll close your PR in about 2 weeks

Cheers
the nf-core maintainers

**Pull request closed without merging**
